pkg/api/types.go

```diff
@@ -93,6 +93,9 @@ var (
 
 // InspectorMetadata is the metadata type with information about image-inspector's operation
 type InspectorMetadata struct {
+	ImageAcquireSuccess bool   // status of aquiring the image
+	ImageAcquireError   string // error message from aquiring the image
```

Comment:
@enoodle I think this should be an `error` that you stringfy only on when some output requires it

Reply:
@simon3z This struct is served as a JSON to the user [1], so I can't have `error` here.

[1] https://github.com/openshift/image-inspector/blob/master/pkg/imageserver/webdav.go#L96

Reply:
never mind  that, it can json marshal `error`, I will make this change.

Reply:
Sorry, apparently it is not possible to meaningfully do it yet
https://github.com/golang/go/issues/5161